### PR TITLE
#4868 [Manager] Create workspace should throw exception if images not exist

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4868)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/store/epics/__tests__/workspace/createWorkspaceEpic.test.js [hunk 1]
```diff
@@ -257,3 +264,174 @@ it('create workspace should be worked correctly', () => {
...
+        payload: {
+          brokerId: bkId,
+        },
+      },
+      x: {
+        type: actions.createWorkspace.FAILURE,
+        payload: {
+          title: 'start broker exceeded max retry count',
+        },
+      },
```
Comment (saivirtue) [inline ohara-manager/client/src/store/epics/__tests__/workspace/createWorkspaceEpic.test.js:423]: As the test result, the final action of failure creation is `CREATE_WORKSPACE/FAILURE`; i.e., in our current version of OharaStream, using **quick create** dialog will see the error event log and the stopped progress of creation

Comment (saivirtue) [reply ohara-manager/client/src/store/epics/__tests__/workspace/createWorkspaceEpic.test.js:423]: /cc @oharastream/developer 
To recover a failed workspace which created by Quick Create Dialog is going to Settings Dialog and click the restart workspace button in Danger Zone 😃

File: ohara-manager/client/src/store/epics/worker/startWorkerEpic.js [hunk 2]
```diff
@@ -63,12 +63,10 @@ export const startWorker$ = params => {
...
-    catchError(err =>
-      from([
-        actions.startWorker.failure(merge(err, { workerId })),
-        actions.createEventLog.trigger({ ...err, type: LOG_LEVEL.error }),
-      ]),
-    ),
+    catchError(res => {
+      // Let the caller decides this Action should be terminated or trigger failure reducer
+      return throwError(res);
+    }),
```
Comment (eechih) [inline ohara-manager/client/src/store/epics/worker/startWorkerEpic.js:69]: 為何要 catchError 不做任何事直接 throwError？是否可以不要 catchError?

Comment (saivirtue) [reply ohara-manager/client/src/store/epics/worker/startWorkerEpic.js:69]: 可以。這裡只是為了讓呼叫的人明確知道**要處理exception的邏輯**

Comment (saivirtue) [reply ohara-manager/client/src/store/epics/worker/startWorkerEpic.js:69]: Maybe add a comment in the function doc will be enough 🤔 
Let me refactor this

Comment (eechih) [reply ohara-manager/client/src/store/epics/worker/startWorkerEpic.js:69]: 3Q~

**Pull request merged**